From 948a96d710f377fb0ed8d78c7fb85b299c80ef13 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Sat, 8 Nov 2014 10:12:54 -0300
Subject: [PATCH] arptables: disable dlfcn.h include

The relevant code is disabled in libarptc_incl.c so it's not required
and breaks the ability to build it in a pure-static toolchain.

Status: sent upstream.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 arptables.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arptables.c b/arptables.c
index 64ac3aa..fe270bc 100644
--- a/arptables.c
+++ b/arptables.c
@@ -35,7 +35,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <inttypes.h>
+#if 0
 #include <dlfcn.h>
+#endif
 #include <ctype.h>
 #include <stdarg.h>
 #include <limits.h>
-- 
2.0.4

